java.hints/src/org/netbeans/modules/java/hints/suggestions/Lambda.java

```diff
@@ -91,6 +92,9 @@
  */
 public class Lambda {
     
+    private static final Set<String> ERROR_CODES = new HashSet<String>(Arrays.asList(
```

Comment:
No need to construct `Set` bcs the`TreeUtilities` method accepts `Collection`. See also review comments for `TreeUtilities` file.

Reply:
Use Array of String in place of Set.